python/test_EventStore.py

```diff
@@ -2,9 +2,17 @@
 
 from EventStore import EventStore
 
+#pylint: disable=no-member # pylint does not realize that the store is populated by inheriting classes
 
 class EventStoreBaseTestCase:
```

Comment:
Called it a `Mixin`, and pylint will know.
https://pylint.pycqa.org/en/2.5/faq.html#how-do-i-avoid-access-to-undefined-member-messages-in-my-mixin-classes